packages/react-ui-core/src/ToggleButton/index.js

```diff
@@ -0,0 +1 @@
+export { default as ToggleButton } from './ToggleButton'
```

Comment:
Please export like everything else is exported:
```
export { default } from './ToggleButton'
export { default as ToggleButton } from './ToggleButton'
```

Also, this probably belongs in `react-ui-core/src/Button` directory.

Reply:
Fixed